scripts/fill_dcp1_file_metadata/README.md

```diff
@@ -0,0 +1,56 @@
+# Exporting a DCP1 dataset
+see ticket ebi-ait/dcp-ingest-central#793
+## Fill missing size and fileContentType
+
+This script fixes the file metadata for DCP1 datasets, in which the `size` and `fileContentType` is not filled due to
+being old pieces of metadata. This is one of the fixes needed for exporting a DCP1 project, as the `exporter` needs
+these 2 fields to generate the file descriptors.
+
+Other fixes that may be needed, not covered in this script, are:
+- Setting up checksums (May not actually be needed - Need to check DCP1 project)
+- Migrating project schema to >15.0.0 (Done for most projects)
+
+### Before you start
+
+- Create a virtual environment: `virtualenv <environment_name>`
```

Comment:
Should the virtual env be created in the script's directory `scripts/fill_dcp1_file_metadata`?